embassy-stm32/src/rcc/l4l5.rs

```diff
@@ -52,7 +52,30 @@ pub struct Pll {
     pub divr: Option<PllRDiv>,
 }
 
-/// Clocks configutation
+#[derive(Clone, Copy)]
```

Comment:
please use the enum from the PAC directly in the public API. `pub use crate::pac::rcc::vals::{ Adcsel as AdcClockSource, ... };`

(old code redefined all the enums due to semver concerns about reexporting stuff from the PAC, but with time we've seen it's hard to maintain because these enums vary a lot across families, so the benefit doesn't outweigh the costs)

Reply:
Fixed